R/plugin-minimap.R

```diff
@@ -14,60 +16,109 @@ leafletMiniMapDependencies <- function() {
-#' Can be positive or negative, defaults to -5.
-#' @param zoomLevelFixed  Overrides the offset to apply a fixed zoom level to the minimap regardless of the main map zoom.
-#' Set it to any valid zoom level, if unset zoomLevelOffset is used instead.
+#' @param collapsedWidth The width of the toggle marker and the minimap
+#'    when collapsed, in pixels. Defaults to 19.
+#' @param collapsedHeight The height of the toggle marker and the minimap
+#'    when collapsed, in pixels. Defaults to 19.
+#' @param zoomLevelOffset The offset applied to the zoom in the minimap compared
+#'    to the zoom of the main map. Can be positive or negative, defaults to -5.
+#' @param zoomLevelFixed  Overrides the offset to apply a fixed zoom level to
+#'    the minimap regardless of the main map zoom.
+#'    Set it to any valid zoom level, if unset zoomLevelOffset is used instead.
+#' @param centerFixed Applies a fixed position to the minimap regardless of
+#'    the main map's view / position. Prevents panning the minimap, but does
+#'    allow zooming (both in the minimap and the main map).
+#'    If the minimap is zoomed, it will always zoom around the centerFixed point.
+#'    You can pass in a LatLng-equivalent object. Defaults to false.
 #' @param zoomAnimation Sets whether the minimap should have an animated zoom.
-#' (Will cause it to lag a bit after the movement of the main map.) Defaults to false.
+#'    (Will cause it to lag a bit after the movement of the main map.)
+#'    Defaults to false.
 #' @param toggleDisplay Sets whether the minimap should have a button to minimise it.
-#' Defaults to false.
+#'    Defaults to false.
 #' @param autoToggleDisplay Sets whether the minimap should hide automatically,
-#' if the parent map bounds does not fit within the minimap bounds.
-#' Especially useful when 'zoomLevelFixed' is set.
+#'    if the parent map bounds does not fit within the minimap bounds.
+#'    Especially useful when 'zoomLevelFixed' is set.
+#' @param minimized Sets whether the minimap should start in a minimized position.
+#' @param aimingRectOptions Sets the style of the aiming rectangle by passing in
+#'    a Path.Options (\url{http://leafletjs.com/reference.html#path-options}) object.
+#'    (Clickable will always be overridden and set to false.)
+#' @param shadowRectOptions Sets the style of the aiming shadow rectangle by passing in
+#'    a Path.Options (\url{http://leafletjs.com/reference.html#path-options}) object.
+#'    (Clickable will always be overridden and set to false.)
+#' @param strings Overrides the default strings allowing for translation.
+#' @param mapOptions Sets Leaflet options for the MiniMap map.
+#'    It does not override the MiniMap default map options but extends them.
 #' @examples
 #' library(leaflet)
 #'
 #' leaf <- leaflet() %>%
 #'   addTiles() %>%
 #'   addMiniMap()
 #'
+#' @seealso \code{\link{providers}}
 #' @export
 addMiniMap <- function(
   map,
+  tiles = NULL,
```

Comment:
Is it important that this argument be first? This functionality has been on master for a long time, I hate to break backward compatibility.